setup.cfg

```diff
@@ -0,0 +1,36 @@
+[metadata]
+name = redirector
+provides = redirector
+package_name = redirector
+author = STScI
+author_email = help@stsci.edu
```

Comment:
Is this right?

Reply:
I'm not sure, but I see it in other repos.  Happy to change it if you can think of a better option.